### ISSUE TITLE
div vs just li

### ISSUE THREAD

**Issue by yassienAbdillahi** [fac26/week6-fullstack-king#50]
Only li's should be the _direct_ children of a ul (for accessibility reasons).

To fix; just remove the wrapping div and apply that same class to the li instead

https://github.com/fac26/week6-fullstack-king/blob/96517607a088bf1f03037324aa770b406b56cb31/components/FruitItem.js#L17-L18

....

https://github.com/fac26/week6-fullstack-king/blob/96517607a088bf1f03037324aa770b406b56cb31/components/FruitItem.js#L30-L31

**Comment by nataliarusu** [comment]
Thanks
